### PR TITLE
Adding documentation for list

### PR DESCRIPTION
**Pull request by AnomalRoil**
RELEASE_NOTES=[DOCUMENTATION] Document list flags

Signed-off-by: Yolan Romailler <yolan@romailler.ch>

### CODE REVIEW

File: docs/commands/list.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+Note: `list` will not change anything, nor encrypt or decrypt anything.
+
+## Flags
+
+Flag | Aliases | Description
+---- | ------- | -----------
+`--limit value` | `-l value`| Max tree depth (default: -1)
+` --flat `      |` -f`      | Print flat list (default: false)
+` --folders`    | `--fo`    |  Print flat list of folders (default: false)
```
Comment (dominikschulz) [inline docs/commands/list.md:23]: `--fo` is a strange shortcut. Maybe we should rename this to `-d`?

Comment (AnomalRoil) [reply docs/commands/list.md:23]: LGTM, at first I wanted to go with `-ff` "flat folder" and ended up with `-fo` ^^'
I'll change it in another PR.

Comment (dominikschulz) [reply docs/commands/list.md:23]: `-d` for directory make sense to me.

Comment (AnomalRoil) [reply docs/commands/list.md]: done

**Pull request merged**